### ISSUE TITLE
Issues with curl in Dockerfile

### ISSUE THREAD

**Issue by GreenfishK** [GreenfishK/BEAR#11]
```
RUN apt-get install curl=7.74.0-1.3+deb11u3 -y
```
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 curl : Depends: libcurl4 (= 7.74.0-1.3+deb11u3) but 7.74.0-1.3+deb11u5 is to be installed

**Comment by GreenfishK** [comment]
Fix by changing to curl=7.74.0-1.3+deb11u5

**Comment by GreenfishK** [comment]
issue appeared again while building the docker image:

```
The command '/bin/sh -c apt-get install curl=7.74.0-1.3+deb11u5 -y' returned a non-zero code: 100
```

**Comment by GreenfishK** [comment]
Fix by changing to curl=7.74.0-1.3+deb11u7
